clang-format-changed-files.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# This script runs clang_format on all .h, .m, and .mm files that have changes from develop branch.
+# Before running this, ensure that your local develop branch is up to date.
+git fetch
+i=0
+for modified_file in $(git diff origin/develop --name-only *.h *.m *.mm)
```

Comment:
On forks origin/develop will likely be stale, while upstream/develop might be the right one to serve as a reference (and could be another name).

This makes this script highly unreliable. We might need to adapt logic to find the right remote to fetch and compare against.

Reply:
Would do: UPSTREAM=`git remote -v | grep ":Microsoft" | awk '{print $1}' | uniq`
git fetch $UPSTREAM
git diff "$UPSTREAM/develop"